react/features/toolbox/components/Toolbox.native.js

```diff
@@ -284,20 +283,6 @@ class Toolbox extends Component<Props> {
     }
 }
 
-/**
- * Additional properties for various icons, which are now platform-dependent.
- * This is done to have common logic of generating styles for web and native.
- * TODO As soon as we have common font sets for web and native, this will no
- * longer be required.
- */
-// $FlowExpectedError
-Object.assign(Toolbox.prototype, {
```

Comment:
I could remove this (yay!) by adding some hack on ToolboxItem.native.js, please take a look there.